templates/terraform.tfvars.aws-template

```diff
@@ -39,3 +39,5 @@ node_disk_size = "20" # Size in GB
 # Cloudflare proxy (optional)
 # cloudflare_proxied = "true"
 # cloudflare_record_texts = ["record1","record2",...] # Warn: wildcards are not supported when using proxied records
+
+provider = "aws"
```

Comment:
why do we need this? It should be in the beginning IMO

Reply:
We need it because then the user doesn't need to keep track of cloud provider when doing apply or destroy etc.
I moved it to the top instead

Reply:
(I am not entirely sure it is better there in top instead of bottom, since it doesn't need to be changed:-)